papermill_origami/noteable_dagstermill/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""This module combines Dagstermill with Noteable.
+
+To accomplish this a new `noteable-dagstermill` engine is created.
+This engine combines the DagstermillEngine with the NoteableEngine.
+To use this engine and module, ensure you install the `dagster` extra.
+"""
+from papermill.engines import papermill_engines
+from papermill.translators import papermill_translators
+
+from .engine import NoteableDagstermillEngine
+from .translator import DagsterTranslator
+
+papermill_engines.register("noteable-dagstermill", NoteableDagstermillEngine)
+papermill_translators.register("python", DagsterTranslator)
```

Comment:
I would put a comment that the import above needs to happen first so that we overwrite the registration they initially do